numba_dpex/dpnp_iface/_intrinsic.py

```diff
@@ -1077,3 +1078,59 @@ def codegen(context, builder, sig, args):
         return ary._getvalue()
 
     return signature, codegen
+
+
+@intrinsic
+def ol_dpnp_nd_array_sycl_queue(
+    ty_context,
+    ty_dpnp_nd_array: DpnpNdArray,
+):
+    if not isinstance(ty_dpnp_nd_array, DpnpNdArray):
```

Comment:
When can this happen? Should a TypingError be raised here?

Reply:
It may happen if user calls it directly. It can not happen if called from overload

Reply:
Added error TypeError raise